src/posts/introducing-pint-xarray/index.md

```diff
@@ -0,0 +1,292 @@
+
+```ipython
+In [2]: distance = xr.DataArray(10).pint.quantify("metres")
+   ...: time = xr.DataArray(4).pint.quantify("seconds")
+   ...:
+   ...: distance / time
+Out[2]:
+<xarray.DataArray ()>
+<Quantity(2.5, 'meter / second')>
+```
+
+## Units are integral to science
+
+All quantities in science have units, whether explicitly or implicitly. (And even dimensionless quantities like ratios still technically have units.)
+
+Getting our units right is finicky, and can very easily go unnoticed in our code.
+
+Even worse, the consequences of getting units wrong can be huge!
+
+The most famous example of a units error has to be NASA's $125 million [Mars Climate Orbiter](https://www.simscale.com/blog/2017/12/nasa-mars-climate-orbiter-metric/), which in 1999 burned up in the Martian atmosphere instead of successfully entering orbit around Mars.
+A trajectory course correction had gone wrong, and the error was eventually traced back to a units mismatch: the engineers at Lockheed Martin expressed impulse in [pound-force](<https://en.wikipedia.org/wiki/Pound_(force)>) seconds, whereas the engineers at JPL assumed the impulse value their part of the software received was in SI newton seconds.
+
+<p align = "center">
+  <img src = "https://clqtg10snjb14i85u49wifbv-wpengine.netdna-ssl.com/wp-content/uploads/2017/12/Customers.jpg" />
+</p>
+
+<p align = "center">
+  Newspaper cartoon depicting the incongruence in the units used by NASA and Lockheed Martin scientists that led to the Mars Climate Orbiter disaster.
+</p>
+
+We should take stories like this seriously: If we can automatically track units we can potentially eliminate a whole class of possible errors in our scientific work...
+
+## Pint tracks units
+
+There are a few packages for handling units in python (notably [unyt](https://github.com/yt-project/unyt) and [astropy.units](https://docs.astropy.org/en/stable/units/)), but for technical reasons we began units integration in xarray with [pint](https://pint.readthedocs.io/en/stable/).
+These various packages work by providing a numerical array type that acts similarly to a numpy array, and is intended to plug in and replace the raw numpy array (a so-called "duck array type").
+
+Pint provides the `Quantity` object, which is a normal numpy array combined with a `pint.Unit`:
+
+```python
+q = np.array([6, 7]) * pint.Unit('metres')
+print(repr(q))
+```
+
+```
+Out: <Quantity([6 7], 'meter')>
+```
+
+Pint Quantities act like numpy arrays, except that the units are carried around with the arrays, propagated through operations, and checked during operations involving multiple quantities.
+
+## Xarray now wraps Pint
+
+Thanks to the [tireless work](https://github.com/pydata/xarray/issues/3594) of xarray core developer Justus Magin, you can now enjoy this automatic unit-handling in xarray!
```

Comment:
```suggestion
Thanks to the [tireless work](https://github.com/pydata/xarray/issues/3594) of Xarray core developer [Justus Magin](https://github.com/keewis), you can now enjoy this automatic unit-handling in Xarray!
```

Reply:
I also think you don't really need to change this either. It's grammatically a bit weird to speak about yourself in the third person, but this way you get due credit for your work!